packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -225,6 +254,49 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
       pointerSignalResolver.resolve(event);
     }
   }
+
+  void _scheduleFrameCallback() {
+    if (_frameCallbackScheduled) {
+      return;
+    }
+    final SchedulerBinding? instance = scheduler ??= SchedulerBinding.instance;
+    assert(instance != null);
+    if (instance != null) {
+      _frameCallbackScheduled = true;
+      instance.scheduleFrameCallback((_) {
+        _frameCallbackScheduled = false;
+        _sampleTime = instance.currentSystemFrameTimeStamp + samplingOffset;
+        if (!locked) {
+          _flushPointerEventQueue();
+        }
+      });
+    }
+  }
+
+  // Flag to track if a frame callback has been scheduled.
+  bool _frameCallbackScheduled = false;
+
+  // Current sample time for resampling.
+  Duration _sampleTime = const Duration();
```

Comment:
`Duration.zero` is probably easier to read than `const Duration();`. So is many following.

Reply:
Done